ote/src/cljs/ote/views/own_services.cljs

```diff
@@ -103,23 +131,73 @@
            [delete-service-action e! row]]])
        services))])
 
-(defn transport-services-listing [e! transport-operator-id services section-label]
+(defn- route-error
+  [errors info-open?]
+  (let [error-keys (reduce
+                     (fn [set error]
+                       (conj set (:error error)))
+                     #{}
+                     errors)]
+    [info/info-toggle
+     (tr [:own-services-page :missing-service-info])
+     [:div
+      [:h5 {:style {:margin-top 0}} (tr [:own-services-page :flaws])]
+      (doall
+        (for [error errors]
+          ^{:key (:name error)}
```

Comment:
Could key be the same if there is same error in two different services?

Reply:
It's the name of the service not the error. Changed the key to `:service-name`. At this moment there can't be more than 1 error per service.